.github/workflows/test.yml

```diff
@@ -46,7 +46,7 @@ jobs:
             go.sum
       - name: start localnet
         run: |
-          make clean localnet-start
+          TESTNET_NVAL=6 make clean localnet-start
         if: env.GIT_DIFF
```

Comment:
recommend removing all of these conditionals.  They're an anti-pattern that caused a number of CI issues on cosmos chains over the past two years.

Reply:
Do you mean `if: env.GIT_DIFF`? You know, I am kind of struggling with that statement: Does it mean that the workflow is only triggered if there is actually a git diff present? I mean... Makes sense to remove that.

Reply:
exactly so sir.  I've removed it all around the cosmos. 

Reason is, the tests are there for running, and the reason they never ran, is those conditionals.